### ISSUE TITLE
Round out the Fund model

### ISSUE THREAD

**Issue by lomky** [DARIAEngineering/dcaf_case_management#2410]
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**
 - Create a Fund data edit page, restricted to admin editing

* **What feature or behavior is this required for?**
Create a fully fleshed out Fund model - work initially started in #2403 

* **How could we solve this issue? (Not knowing is okay!)**
Pretty standard rails, methinks?

* **Anything else?**
@colinxfleming to add any more details or split this up!

**Comment by colinxfleming** [comment]
yep, let's do this! Some fields on this model are sacred but some are safe to change. I think roughly:

* in its own controller/view, permissioned to admin only (Configs is a safe approach to copy)
* accessible thru the admin dropdown in the toolbar
<img width="263" alt="image" src="https://user-images.githubusercontent.com/3866868/213885912-5a1d3176-5c4a-4dd0-9139-6b54080fdba6.png">
* let's have the URLs be GET /funds/edit for the form view, and PUT/PATCH /funds for the update action (similar to how `users` actions are set up). This is because a user session should be scoped to a particular fund as is and there's never a reason to have multiple-fund access like that. 
* fields that should be editable: full_name, site_domain, phone

**Comment by cameron-hicks** [comment]
Starting on this! Pairing welcome.
